native/src/components/MapView.tsx

```diff
@@ -46,14 +50,28 @@ const MapView = ({ boundingBox, featureCollection }: MapViewPropsType): ReactEle
     zoomLevel: defaultViewportConfig.zoom,
     bounds
   }
+
+  const updateLocationPermission = useCallback((locationPermission: boolean) => {
+    setLocationPermission(locationPermission)
+    setFollowUserLocation(false)
+    if (locationPermission) {
```

Comment:
i would prefer ternary expression shorthand if only one line of code follows
locationPermission && setFollowUserLocation(true)